prime-router/src/test/kotlin/fhirengine/engine/LookupTableValueSetTests.kt

```diff
@@ -0,0 +1,113 @@
+package gov.cdc.prime.router.fhirengine.engine
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFailure
+import assertk.assertions.isTrue
+import gov.cdc.prime.router.Metadata
+import gov.cdc.prime.router.fhirengine.translation.hl7.FhirTransformer
+import gov.cdc.prime.router.fhirengine.translation.hl7.schema.fhirTransform.FhirTransformSchema
+import gov.cdc.prime.router.fhirengine.translation.hl7.schema.fhirTransform.FhirTransformSchemaElement
+import gov.cdc.prime.router.fhirengine.translation.hl7.schema.fhirTransform.fhirTransformSchemaFromFile
+import gov.cdc.prime.router.metadata.LookupTable
+import gov.cdc.prime.router.unittest.UnitTestUtils
+import io.mockk.every
+import io.mockk.mockkConstructor
+import io.mockk.mockkObject
+import io.mockk.unmockkAll
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.HumanName
+import org.hl7.fhir.r4.model.Patient
+import org.hl7.fhir.r4.model.StringType
+import org.junit.jupiter.api.Test
+import tech.tablesaw.api.StringColumn
+import tech.tablesaw.api.Table
+
+class LookupTableValueSetTests {
+    @Test
+    fun `test read extended FHIR Transform from file`() {
+        val testTable = Table.create(
+            "lookuptable",
+            StringColumn.create("key", "abc123", "def456"),
+            StringColumn.create("value", "ghi789", "")
+        )
+        val testLookupTable = LookupTable(name = "lookuptable", table = testTable)
+
+        mockkConstructor(Metadata::class)
+        every { anyConstructed<Metadata>().findLookupTable("lookuptable") } returns testLookupTable
+        mockkObject(Metadata)
```

Comment:
I think you can drop this and line 39

Reply:
The test should function either way but I wanted to short circuit this so that the test could run quicker. `Metadata` has its own suite of tests and we don't need it to go to the database to load lookup tables when we won't be using the actual lookup table information.